catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/authentication/kerberos/KerberosClient.java

```diff
@@ -63,6 +63,12 @@ public String login(String keytabFilePath) throws IOException {
     Preconditions.checkArgument(
         principalComponents.size() == 2, "The principal has the wrong format");
 
+    // Set Kerberos krb file
+    String krbFilePath = kerberosConfig.getKrb5Conf();
+    Preconditions.checkArgument(
+        StringUtils.isNotBlank(krbFilePath), "The Kerberos krb file can't be blank");
+    System.setProperty("java.security.krb5.conf", krbFilePath);
```

Comment:
Is this method class loader isolated? I mean what if I set `java.security.krb5.conf` in `catalogA`, does it take effect  in `catalogB`

Reply:
The configuration is separated for each catalog, this mean the setting of catalogA can not affect catalogB.

Reply:
I mean if we set the value in `catalogA`, and in catalogB,  could we still get the value if `catalogB` does not set it?